scripts/docker/restart_puma.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# simple script to hot restart puma, https://github.com/puma/puma#restart
+# e.g. docker exec -it 'panoptes_container_name' bash -c "/rails_app/restart_puma.sh"
+kill -s SIGUSR2 `ps aux | grep [p]uma | awk '{print $2}'`
```

Comment:
The containers should have pkill, which would make it as simple as `pkill -USR2 puma`.